src/index.js

```diff
@@ -19,7 +21,7 @@ class Marquee extends React.Component {
   }
 
   getListingFromServer() {
-    axios.get('/listing/5fffa8a5a2e0728842416112')
+    axios.get('/listing/600a6b89cac7839e11ac6fba')
```

Comment:
In your axios call, be sure you call the link: 'http://localhost:PORTNUM/listing/......'
This will ensure the request goes to the correct server when being served up in the proxy.